tests/unit/models/builder-test.js

```diff
@@ -498,4 +502,42 @@ describe('models/builder.js', function() {
       });
     });
   });
+
+  if (isExperimentEnabled('BROCCOLI_2')) {
+    describe('fallback from broccoli 2 to broccoli-builder', function() {
+      it('falls back to broccoli-builder if an InvalidNode error is thrown for read/rebuild api', function() {
+        let spy = sinon.spy(console, 'warn');
+
+        const builder = new Builder({
```

Comment:
This instance of the builder won’t have `this.ui` because it is not being constructed with it.